### PR TITLE
Feat add ps script

### PR DESCRIPTION
**Pull request by ColJurten**
new ps test

### CODE REVIEW

**Comment by ColJurten** [comment]
can you look at this pull request

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # git_training
 Repository to train coachees to use git
-bannana
\ No newline at end of file
+text test
```
Comment (Khalshim) [inline README.md]: Is this line really useful ?

File: test.ps1 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# create a script that will write hello world to the console
```
Comment (Khalshim) [inline test.ps1]: Change this githubcopilot command into a proper commentary

**Review by Khalshim — CHANGES_REQUESTED** [review]
Please modify your files accordingly to comment

**Review by Khalshim — APPROVED** [review]
let's ship it

**Pull request merged**